nilearn/plotting/displays/_slicers.py

```diff
@@ -455,17 +514,19 @@ def _show_colorbar(self, cmap, norm, cbar_vmin=None,
         _, y0, x1, y1 = self.rect
         height = y1 - y0
         x_adjusted_width = self._colorbar_width / len(self.axes)
-        x_adjusted_margin = self._colorbar_margin['right'] / len(self.axes)
-        lt_wid_top_ht = [x1 - (x_adjusted_width + x_adjusted_margin),
-                         y0 + self._colorbar_margin['top'],
-                         x_adjusted_width,
-                         height - (self._colorbar_margin['top'] +
-                                   self._colorbar_margin['bottom'])]
+        x_adjusted_margin = self._colorbar_margin["right"] / len(self.axes)
+        lt_wid_top_ht = [
+            x1 - (x_adjusted_width + x_adjusted_margin),
+            y0 + self._colorbar_margin["top"],
+            x_adjusted_width,
+            height
+            - (self._colorbar_margin["top"] + self._colorbar_margin["bottom"]),
+        ]
         self._colorbar_ax = figure.add_axes(lt_wid_top_ht)
-        if _compare_version(matplotlib.__version__, '>=', "1.6"):
-            self._colorbar_ax.set_facecolor('w')
+        if _compare_version(matplotlib.__version__, ">=", "1.6"):
```

Comment:
Note to self: check where `_compare_version` is used in codebase referencing package versions we don't support anymore and remove extraneous code

Will open an issue